OpenDreamRuntime/Objects/MetaObjects/DreamMetaObjectClient.cs

```diff
@@ -98,6 +100,14 @@ sealed class DreamMetaObjectClient : IDreamMetaObject {
                 case "key":
                 case "ckey":
                     return new(_dreamManager.GetSessionFromClient(dreamObject).Name);
+                case "computer_id": // FIXME: This is not secure! Whenever RT implements a more robust (heh) method of uniquely identifying computers, replace this impl with that.
+                    MD5 md5 = MD5.Create();
+                    /// <remarks>Check on <see cref="Robust.Shared.Network.NetUserData.HWId"/> if you want to seed from how RT does user identification.
+                    /// We don't use it here because it is probably not enough to ensure security, and (as of time of writing) only works on Windows machines.</remarks>
+                    byte[] brown = Encoding.UTF8.GetBytes(_dreamManager.GetSessionFromClient(dreamObject).Name);
+                    byte[] hash = md5.ComputeHash(brown);
+                    string hashStr = BitConverter.ToString(hash).Replace("-", "").ToLower();
+                    return new(long.Parse(hashStr, System.Globalization.NumberStyles.HexNumber).ToString()); // Converts from hex to decimal. Output is in analogous format to BYOND's.
```

Comment:
This is throwing an exception on me due to `hashStr` being too large for an Int64 (with the username `JoeGenero2`)

Reply:
😒 forgot C# has overflow exceptions